networkx/algorithms/bipartite/extendability.py

```diff
@@ -0,0 +1,158 @@
+""" Provides a function for computing the extendability of
+graph which is undirected, simple, connected and bipartite.
```

Comment:
You should impose the constraint that the input graph be undirected and simple using 
```python
@not_implemented_for("directed")
@not_implemented_for("multigraph")
def find_extendability(G):
```

Then remove the tests at the top of the function that test the type of graph.
```